hoods.php

```diff
@@ -17,28 +17,29 @@
 $hoods = array();
 while ( $result = $rs->fetch ( PDO::FETCH_ASSOC ) ) {
 	$hood = array();
-	$hood['id']   = intval($result['ID']);
+	$hood['id'] = intval($result['ID']);
 	$hood['name'] = $result['name'];
-	$hood['net']  = $result['net'];
+	$hood['net'] = $result['net'];
 	$hood['essid_ap'] = $result['ESSID_AP'];
+	$hood['active'] = $result['active'];
 	if ($result ['lat'] > 0 && $result ['lon'] > 0) {
 		$hood['lat'] = floatval($result['lat']);
 		$hood['lon'] = floatval($result['lon']);
 	}
 
-	if(isset($polydata[$result['ID']])) {
+	if (isset($polydata[$result['ID']])) {
```

Comment:
Sowas macht man normalerweise nicht, da das Stiländerungen sind. Wenn man das möchte, dann in einem separaten Patch für den ganzen File.